README.MD

```diff
@@ -93,6 +93,55 @@ Navigate to the folder `infrastructure/local` directory within the clone of the
 
 Execute `docker compose down` to stop the infrastructure containers
 
+# Local Deployment with java -jar
+
+## Start PostgreSQL, Kafka and Zookeeper
+
+This starts them as Docker containers.
+
+    cd run-locally
+    docker-compose up -d
+    cd ..
+
+## Build jar
+
+    mvn clean package -DskipTests
+
+## Set environment variables
+
+    export DB_DATABASE=folio
+    export DB_PASSWORD=folio
+    export DB_USERNAME=folio
+    export KAFKA_PORT=29092
```

Comment:
Why is any of this needed? 

It seems to directly contradict the instructions provided above for running a module locally using Docker.

Reply:
I've added an introduction that explains the differences between the two instructions.

Both have their con: Additional docker build step versus environment variables in the shell.

I've added the env variables to my .bashrc to avoid the additional docker build step.

If you don't like the "java -jar without container" instructions being mentioned in mod-inventory-storage repository I can put them at some other place like dev.folio.org or a new subdirectory of https://github.com/folio-org/folio-tools/tree/master/infrastructure .

Reply:
Why does your new approach require a new docker compose file at all? 

There is already a docker compose file that can be used to set up infrastructure that can be used with a module deployed either with docker or as a jar. 

Why replicate that in every module repository?

Reply:
> If you don't like the "java -jar without container" instructions being mentioned in mod-inventory-storage repository

I'm more than ok with having instructions for how to run the module as a jar locally.

I think it would be better if those instructions:
* used what we already have e.g. use the docker compose file for spinning up infrastructure rather than defining a new one
* are added alongside a review of the existing documentation we have for running the module locally (which uses Okapi and some scripts and may well be out of date or redundant)